### PR TITLE
CIV-3562 N245 Form screen

### PR DESCRIPTION
**Pull request by karthick-mohanasundaram-HMCTS**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-3562

### Change description ###
Create Application to Vary Judgement

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: ccd-definition/ComplexTypes/GAN245FormUpload.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "ID": "GAN245FormUpload",
+    "ListElementCode": "generalAppN245Download",
+    "FieldType": "Label",
+    "ElementLabel": "<p class=\"govuk-inset-text\">if you have not already completed it, <a href=\"https://www.gov.uk/government/publications/form-n245-application-for-suspension-of-a-warrant-and-or-variation-of-an-order#:~:text=Form-,Form%20N245%3A%20Apply%20to%20suspend%20a%20warrant%20or%20vary%20payments,warrant%20issued%20by%20a%20court\" rel=\"noreferrer noopener\" target=\"_blank\">download the N245 form.</a>.</p>",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "GAN245FormUpload",
+    "ListElementCode": "generalAppN245Label",
+    "ElementLabel": "<br/>When you upload the form, include the case number in the file name.\n\nIf you have printed off the form and filled it in by hand, make sure the size of your saved file is no more than 100MB.\n\nSave your file as either: JPG, PNG, SVG, GIF, DOC, DOCX or PDF",
```
Comment (MMNycz) [inline ccd-definition/ComplexTypes/GAN245FormUpload.json:12]: Karthick - just to make sure - we want to leave it like that on screen for now?

Comment (karthick-mohanasundaram-HMCTS) [reply ccd-definition/ComplexTypes/GAN245FormUpload.json:12]: Nope, it should be another ticket to configure the screen

**Review by MMNycz — APPROVED** [review]

**Review by pliao-hmcts — APPROVED** [review]

**Review by deepthidoppalapudihmcts — COMMENTED** [review]
I think file name needs to end with GASpec.. Isn't it?

**Comment by karthick-mohanasundaram-HMCTS** [comment]
> I think file name needs to end with GASpec.. Isn't it?

Done

**Pull request merged**